.github/workflows/ci.yaml

```diff
@@ -949,7 +949,7 @@ jobs:
           tool: cargo-codspeed
 
       - name: "Build benchmarks"
-        run: cargo codspeed build --features "codspeed,instrumented" --no-default-features -p ruff_benchmark --bench formatter --bench lexer --bench linter --bench parser
+        run: cargo codspeed build --features "codspeed,instrumented" --no-default-features --profile profiling -p ruff_benchmark --bench formatter --bench lexer --bench linter --bench parser
```

Comment:
You probably want to change the profile for other jobs too? Because this is only for the formatter.